src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockCwdsDAO.scala

```diff
@@ -1,34 +1,74 @@
 package org.broadinstitute.dsde.firecloud.dataaccess
 
+import akka.http.scaladsl.model.StatusCodes.{BadRequest, EnhanceYourCalm, Forbidden, UnavailableForLegalReasons}
+import org.broadinstitute.dsde.firecloud.dataaccess.LegacyFileTypes.{FILETYPE_PFB, FILETYPE_RAWLS, FILETYPE_TDR}
 import org.broadinstitute.dsde.firecloud.model.{AsyncImportRequest, ImportServiceListResponse, UserInfo}
+import org.broadinstitute.dsde.rawls.model.ErrorReportSource
+import org.databiosphere.workspacedata.client.ApiException
 import org.databiosphere.workspacedata.model.GenericJob
 import org.databiosphere.workspacedata.model.GenericJob.{JobTypeEnum, StatusEnum}
 
 import java.time.OffsetDateTime
 import java.util.UUID
 
-class MockCwdsDAO(enabled: Boolean = true) extends CwdsDAO {
+class MockCwdsDAO(
+    enabled: Boolean = true,
+    supportedFormats: List[String] = List("pfb", "tdrexport", "rawlsjson")
+) extends HttpCwdsDAO(enabled, supportedFormats) {
+  implicit val errorReportSource: ErrorReportSource = ErrorReportSource(
+    "MockCWDS"
+  )
+  override def listJobsV1(workspaceId: String, runningOnly: Boolean)(implicit
+      userInfo: UserInfo
+  ): List[ImportServiceListResponse] = List()
 
-  override def isEnabled: Boolean = enabled
-
-  override def getSupportedFormats: List[String] = List("pfb", "tdrexport", "rawlsjson")
-  override def listJobsV1(workspaceId: String, runningOnly: Boolean)(implicit userInfo: UserInfo)
-  : List[ImportServiceListResponse] = List()
-
-  override def getJobV1(workspaceId: String, jobId: String)(implicit userInfo: UserInfo): ImportServiceListResponse =
+  override def getJobV1(workspaceId: String, jobId: String)(implicit
+      userInfo: UserInfo
+  ): ImportServiceListResponse =
     ImportServiceListResponse(jobId, "ReadyForUpsert", "pfb", None)
 
-  override def importV1(workspaceId: String,
-                        asyncImportRequest: AsyncImportRequest
-                       )(implicit userInfo: UserInfo): GenericJob = {
+  override def importV1(
+      workspaceId: String,
+      importRequest: AsyncImportRequest
+  )(implicit userInfo: UserInfo): GenericJob = {
+    importRequest.filetype match {
```

Comment:
Copied this logic over from `MockImportServiceDAO` and then swapped this in wherever tests relied on it when interacting with `ImportService`.